compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -946,7 +945,10 @@ pub trait PrettyPrinter<'tcx>:
         &mut self,
         trait_ref: ty::PolyTraitRef<'tcx>,
         proj_ty: Option<(DefId, ty::Binder<'tcx, Ty<'tcx>>)>,
-        traits: &mut FxHashMap<ty::PolyTraitRef<'tcx>, BTreeMap<DefId, ty::Binder<'tcx, Ty<'tcx>>>>,
+        traits: &mut FxHashMap<
```

Comment:
Where is this FxHashMap used? Does any code depend on the iteration order?

Reply:
This map is used in two places:

- https://github.com/rust-lang/rust/blob/181e91567c9f347e055b33b1d7e9894f769aafe3/compiler/rustc_middle/src/ty/print/pretty.rs#L797
- https://github.com/rust-lang/rust/blob/181e91567c9f347e055b33b1d7e9894f769aafe3/compiler/rustc_middle/src/ty/print/pretty.rs#L806-L807